component/index.js

```diff
@@ -1,4 +1,9 @@
 const sideNavAnchors = document.querySelectorAll(".side-nav-anchor");
+const snackbarShowBtn = document.querySelector(".snackbar-show-btn");
+const snackbarHideBtn = document.querySelector(".snackbar-hide-btn-demo");
+const snackbar = document.querySelector(".snackbar-demo");
+
+// -------- script for active side navbar link ------- 
 
 for(let i=0 ; i< sideNavAnchors.length;i++){
```

Comment:
Everything looks good. Just one suggestion, you can use `forEach` here instead of using index based for loop. Because, I think, we can use `forEach` on nodelist array.

Reply:
Will try it this way and merge it in next PR